src/index.js

```diff
@@ -81,6 +81,13 @@ export default class Voices {
   polish () {
     return this.byLangCode('pl-PL')
   }
+  indianEnglish () {
+    return this.byLangCode('en-IN')
+  }
+
+  indian () {
+    return this.byLangCode('en-IN')
+  }
```

Comment:
Let's stick with only **indianEnglish** helper method for the Indian English language. Hindi is the only bilingual language in AWS Polly and needs special attention.

You can see the extra property in [voices.json](https://github.com/chgasparoto/aws-polly-voices/blob/master/src/voices.json#L179) file.